README.md

```diff
@@ -16,33 +16,16 @@
   </a>
 </p>
 
-![Hero Image with Logo](./.github/hero2.png)
+![Hero Image with Logo](https://raw.githubusercontent.com/microsoft/react-native-windows/master/.github/hero2.png)
 
-[React Native](https://reactnative.dev) enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and [React](https://reactjs.org/). The focus of React Native is on developer efficiency across all the platforms you care about - learn once, write anywhere. Facebook uses React Native in multiple production apps and will continue investing in React Native.
+> See the official [React Native website](https://facebook.github.io/react-native/) for an introduction to React Native.
```

Comment:
https://reactnative.dev/ here  👍

Reply:
Thanks for catching this! Just pushed the fix.